eogrow/core/area/base.py

```diff
@@ -42,8 +42,8 @@ class BaseAreaManager(EOGrowObject, metaclass=ABCMeta):
     NAME_COLUMN = "eopatch_name"
 
     class Schema(ManagerSchema):
-        patch_list: Optional[PatchListSchema] = Field(
-            description="Names of EOPatches to keep when filtering in `get_names_and_bboxes` method."
+        patch_names_list: Optional[PatchListSchema] = Field(
```

Comment:
I find "list" here redundant, could just be "patch_names".

However, it's OK to have `patch_list` then for the `List[Tuple[str, BBox]]`.

Reply:
i was afraid that `patch_names` carries a connotation that it is a list of 'how to name patches'

to be fair `patch_names_list` is not much better in that regard, we're just used to the `patch_list` name.

Also related to your comment bellow, how do you feel about having this parameter named `relevant_patch_names`?

Reply:
From this point of view, assuming that `patch` means a named bbox `Tuple[str, BBox]`, then it's fine if we use:
- `patch_names` and `relevant_patch_names`/`filtered_patch_names`/`patch_names_subset` for `List[str]`
- `patch_list` for `List[Tuple[str, BBox]]`

I feel the `names` part already suggests a list and is sufficient.

> Also related to your comment bellow, how do you feel about having this parameter named relevant_patch_names?

I'm assuming this is related to [eogrow/core/area/base.py](https://github.com/sentinel-hub/eo-grow/pull/183/files/3330fcd411377dfde1fbafb2ec4f8b0c084d7bf4#diff-a664a1d150adb15d5edbfaeda2b2ffbef2e905d1b9ce5449670f5f88406b1056) L83?

Reply:
I am leaning towards `patch_names_subset` since we have a `test_subset`. Although `relevant_patch_names` sounds nicer and is very clear

Reply:
I went with `relevant_patch_names`, how does the code look to ya?

Reply:
Wait, I must have misunderstood. I didn't realize you meant to use `relevant_patch_names` _instead_ of `patch_names`. My comments for `relevant_*` were for L83. I think that in this case `patch_names` was completely satisfactory, especially together with the description.

Reply:
changed to `patch_names`